update-extension-dependencies.sh

```diff
@@ -5,10 +5,24 @@
 set -e -u -o pipefail
 shopt -s failglob
 
-echo ''
-echo 'Building bom-descriptor-json...'
-echo ''
-mvn -q -e clean package -f devtools/bom-descriptor-json -Denforcer.skip $*
+# CI env var => https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables
+if [ "${CI:-}" == true ]
+then
+  echo ''
+  echo 'Building bom-descriptor-json...'
+  echo ''
+  mvn -q -e clean package -f devtools/bom-descriptor-json -Denforcer.skip $*
+else
+  read -n1 -p 'Did you build the entire project? [y/n] ' ANSWER
+  echo ''
+  if [ "${ANSWER}" != y ]
+  then
+    echo ''
+    echo 'Building entire project...'
+    echo ''
+    mvn -q -e -Dquickly -T0.8C $*
```

Comment:
Some stderr outputs leak through here, e.g. some Gradle stuff.
We could either live with it or print everything. Redirecting stderr to `/dev/null` is not a good idea in case there is a build error.